Userland/Libraries/LibWeb/DOM/Window.cpp

```diff
@@ -202,10 +199,8 @@ i32 Window::request_animation_frame(JS::FunctionObject& js_callback)
 {
     auto callback = request_animation_frame_driver().add([this, handle = JS::make_handle(&js_callback)](i32 id) mutable {
         auto& function = *handle.cell();
-        auto& vm = function.vm();
         (void)JS::call(function.global_object(), function, JS::js_undefined(), JS::Value(performance().now()));
-        if (vm.exception())
-            vm.clear_exception();
+        // FIXME: Do we need to show unhandled exceptions here?
```

Comment:
See https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#run-the-animation-frame-callbacks step 3.3

Reply:
Basically the same as the FIXME in `queue_microtask` below.